### PR TITLE
Collect kube-audit logs if they are avaliable.

### PR DESCRIPTION
**Pull request by granular-ryanbonham**
###### Description

Kube Audit Log Collection

###### Testing performed

Running latest stable release with these changes.

### CODE REVIEW

File: deploy/helm/sumologic/conf/logs/logs.source.kube-audit.conf [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<match kube-audit.**>
+    @type relabel
+    @label @KUBEAUDIT
+</match>
+<label @KUBEAUDIT>
+    <filter kube-audit.**>
+    @type kubernetes_sumologic
```
Comment (rvmiller89) [inline deploy/helm/sumologic/conf/logs/logs.source.kube-audit.conf]: are these lines missing indent?

File: deploy/helm/sumologic/conf/logs/logs.source.kube-audit.conf [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    add_stream "#{ENV['ADD_STREAM']}"
+    add_time "#{ENV['ADD_TIME']}"
+    exclude_namespace_regex "#{ENV['EXCLUDE_NAMESPACE_REGEX']}"
+    </filter>
+    <match **>
+    @type sumologic
+    @id sumologic.endpoint.logs.kubeaudit
+    @include logs.output.conf
+    <buffer>
+        @type memory
```
Comment (rvmiller89) [inline deploy/helm/sumologic/conf/logs/logs.source.kube-audit.conf]: this is missing file-based buffer capability

File: deploy/helm/sumologic/values.yaml [hunk 3]
```diff
@@ -302,7 +302,19 @@ fluent-bit:
...
       Read_From_Tail  true
-
+    [INPUT]
+      Name              tail
+      Path              /var/log/kube-apiserver-audit.log
+      Parser            docker
+      Tag               audit.*
+      Refresh_Interval  5
+      Rotate_Wait       60
+      Mem_Buf_Limit     35MB
```
Comment (rvmiller89) [inline deploy/helm/sumologic/values.yaml]: how come do these values differ from our normal `/var/log/containers/*.log` tail?

File: deploy/helm/sumologic/values.yaml [hunk 4]
```diff
@@ -302,7 +302,19 @@ fluent-bit:
...
+      Buffer_Chunk_Size 2MB
+      Buffer_Max_Size   10MB
+      Skip_Long_Lines   On
+      DB                /tail-db
```
Comment (rvmiller89) [inline deploy/helm/sumologic/values.yaml]: is this right? the other tail input specifies a `.db`.  Also are we missing `DB.Sync Normal` ?

**Comment by granular-ryanbonham** [comment]
Closing in favor of https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/434/files.
@rvmiller89  Ryan I addressed your conserns in the new MR. As for the reason the " values differ from our normal /var/log/containers/*.log tail" I used the kube-audit setting that are default in the upstream fluentbit helm chart.

**Pull request closed without merging**